opentreemap/treemap/views/misc.py

```diff
@@ -71,6 +71,8 @@ def index(request, instance):
 
 
 def get_map_view_context(request, instance):
+    def make_dummy_tree():
```

Comment:
I would throw a comment above this method explaining _why_ the Map page needs a dummy tree variable